src/css/lesson.css

```diff
@@ -0,0 +1,198 @@
+:root {
```

Comment:
need to delete this
all already defined in base.css 
pls use existing common params